### PR TITLE
add io latency example

### PR DESCRIPTION
**Pull request by Andrewmatilde**
Signed-off-by: andrewmatilde <davis6813585853062@outlook.com>
link: https://github.com/Andrewmatilde/tidb-chaos/blob/main/io/latency.md

This is my first example in this repo.Do you have any advice about example of chaos on tidb? PTAL @WangXiangUSTC .

### CODE REVIEW

File: io/latency.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
+# Test IO Chaos on TiDB
+## IO Latency
+
+### Description
+Test the availability of TiDB cluster in IO Chaos with IO latency scenarios.
+
+### Hypothesis
+
+When about one third of TiKV pod in TiDB cluster suffered a high IO latency , the TiDB cluster can still provide services normally.
```
Comment (WangXiangUSTC) [inline io/latency.md]:
```suggestion
When about one third of TiKV pod in TiDB cluster suffered a high IO latency, the TiDB cluster can still provide services normally.
```

File: io/latency.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+1. Install chaos-mesh
+2. A TiDB cluster on k8s with at least 3 tikv POD.
+3. Runing payload on TiDB cluster
+
+### Quick start
+1. Chaos Mesh fault YAML configuration:
+```
+IOChaos-Latency.yaml
+```
+```
```
Comment (WangXiangUSTC) [inline io/latency.md]:
```suggestion
```YAML
```

File: io/latency.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+  containerName: tikv
+```
+2. Using Kubectl to create the experiment:
+```
+kubectl create -f IOChaos-Latency.yaml
+```
+3. Verifying TiDB's status:
+
+    Check QPS & TPS of TiDB in Grafana.
+4. Resault:
```
Comment (WangXiangUSTC) [inline io/latency.md]:
```suggestion
4. Result:
```

File: io/latency.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+  namespace: ${TiDB-Namespace}
+  name: IO-Latency
+spec:
+  selector:
+    namespaces:
+      - ${TiDB-Namespace}
+    pods:
+       ${TiDB-Namespace}:
+        - Some-Of-TiKV-PODS
+  mode: all
```
Comment (WangXiangUSTC) [inline io/latency.md:32]: how about using `fixed-percent`?

File: io/latency.md [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+  volumePath: /var/lib/tikv
+  containerName: tikv
+```
+2. Using Kubectl to create the experiment:
+```
+kubectl create -f IOChaos-Latency.yaml
+```
+3. Verifying TiDB's status:
+
+    Check QPS & TPS of TiDB in Grafana.
```
Comment (WangXiangUSTC) [inline io/latency.md:47]: how about adding a TODO below, we will fill some pictures of Grafana here later.

File: io/latency.md (hunk 1 above)
Comment (Andrewmatilde) [reply io/latency.md]: Fixed.

File: io/latency.md (hunk 2 above)
Comment (Andrewmatilde) [reply io/latency.md]: Fixed.

File: io/latency.md (hunk 3 above)
Comment (Andrewmatilde) [reply io/latency.md]: Fixed.

File: io/latency.md (hunk 4 above)
Comment (Andrewmatilde) [reply io/latency.md:32]: I think in chaos experiment a low randomness may help us to reproduce error, so I delete `percent` field here.

File: io/latency.md (hunk 5 above)
Comment (Andrewmatilde) [reply io/latency.md:47]: Added.

File: io/latency.md (hunk 4 above)
Comment (WangXiangUSTC) [reply io/latency.md:32]: But it is conflict with the `Hypothesis` "When about one third of TiKV pod in TiDB cluster suffered a high IO latency, the TiDB cluster can still provide services normally."

File: io/latency.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+
+Judge whether the hypothesis is correct or not based on the results of the test process.
+
+### More example
+
+In practise, a continues IO latency chaos with some intervals is a nice example.
+
+
+Continues IO Latency intervals:
+```
```
Comment (WangXiangUSTC) [inline io/latency.md]:
```suggestion
```YAML
```

File: io/latency.md (hunk 4 above)
Comment (Andrewmatilde) [reply io/latency.md:32]: I wish users fix the choosed TiKV Pods and choose one third of TiKV pods here.

File: io/latency.md (hunk 6 above)
Comment (Andrewmatilde) [reply io/latency.md]: Fixed.

File: io/latency.md [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+        containerName: tikv
+        methods:
+          - read
+          - write
+    - name: Interval-3
+      templateType: Suspend
+      deadline: 5m
+```
+
+### TiDB Status
```
Comment (WangXiangUSTC) [inline io/latency.md]: what is the purpose of adding the TiDB's status here? 🤔

Comment (Andrewmatilde) [reply io/latency.md]: I think this may be meaningless now , I will delete it.

**Review by WangXiangUSTC — APPROVED** [review]
LGTM

**Comment by WangXiangUSTC** [comment]
@cwen0 PTAL

**Review by cwen0 — APPROVED** [review]
LGTM

**Pull request merged**